Podfile

```diff
@@ -1,4 +1,4 @@
-LP_VERSION = ENV['LP_VERSION'] ? ENV['LP_VERSION'] : '2.0.5'
+LP_VERSION = ENV['LP_VERSION'] ? ENV['LP_VERSION'] : '2.0.6-rc.1'
```

Comment:
this can just be 2.0.6 now, no rc1 required

Reply:
done